modules/configuring-firewall.adoc

```diff
@@ -48,27 +48,18 @@ Before you install {product-title}, you must configure your firewall to grant ac
 |===
 |Cloud |URL |Function
 
-.2+|AWS
-
-|`route53.amazonaws.com`
-|Allows the Cluster Ingress Operator to creates wildcard alias records for routes
-
-|`amazonaws.com`
-|Required to access AWS services and resources, including s3, iam, and `quay-registry.s3`
+|AWS
+|`*.amazonaws.com`
+|Required to access AWS services and resources, including s3, iam, and `quay-registry.s3`. Review the link:https://docs.aws.amazon.com/general/latest/gr/rande.html[AWS Service Endpoints] in the AWS documentation to determine the exact endpoints to allow for the regions that you use.
```

Comment:
I would lose the ", including s3, iam, and `quay-registry.s3` bit. quay-registry is called out as a separate row. Also, identifying just S3 and IAM is not all the services we use. Those are identified in the AWS portion of the docs if they want to get real specific on endpoints.